Resources/Prototypes/_Goobstation/Entities/Objects/Devices/Electronics/misc_signallers.yml

```
@@ -0,0 +1,71 @@
+    receiveFrequencyId: BasicDevice
+  - type: WirelessNetworkConnection
+    range: 40
+
+# voice signaller
+- type: entity
+  parent: MiscSignallerBase
+  id: VoiceSignaller
+  name: voice signaller
+  description: A device that turns vocal percussion into signals, straight from your mind.
+  components:
+   - type: Sprite
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 3

(indentation)